marconi-mamba/examples/json-rpc-server/src/Main.hs

```diff
@@ -33,7 +33,7 @@ cliParser = CliOptions
     <$> strOption (long "utxo-db"
                               <> short 'd'
                               <> metavar "FILENAME"
-                              <> help "Path to the utxo SQLite database.")
+                              <> help "Path to the marconi SQLite database.")
```

Comment:
That was changed when we changed the CLI to provide a directory path where we create the individual database files , rather than path for individual files for each database. However, the wording is confusing, I'll reword it, thanks